static/redfish/v1/$metadata/index.xml

```diff
@@ -213,13 +213,15 @@
     <edmx:Reference Uri="/redfish/v1/schema/Battery_v1.xml">
         <edmx:Include Namespace="Battery"/>
         <edmx:Include Namespace="Battery.v1_0_0"/>
+        <edmx:Include Namespace="Battery.v1_0_1"/>
```

Comment:
Ali mentioned this is just from running the script, apparently update_schema was updated but not run

Reply:
I will move additions of update_schemas.py to a separate script. I think I should only include the relevant changes to ACF

Reply:
I moved these changes into a separate commit in this PR.

Reply:
Done.